src/patient-registration/patient-registration-helper.tsx

```diff
@@ -41,6 +32,6 @@ export type Patient = {
     birthdate: Date;
     birthdateEstimated: boolean;
     attributes: Array<AttributeValue>;
-    addresses: Array<AddressValue>;
+    addresses: Array<any>;
```

Comment:
Is this really any? I would say it may be rather `Record<string, string>` or something similar, right?